utils/random.go

```diff
@@ -0,0 +1,41 @@
+package utils
+
+import (
+	"math/rand"
+	"strings"
+	"time"
+)
+
+const alphabet = "abcdefghijklmnopqrstuvwxyz"
+
+func init() {
+	rand.Seed(time.Now().UnixNano())
+}
+
+// RandomInt generates a random integer between min and max
+func RandomInt(min, max int64) int64 {
+	return min + rand.Int63n(max-min+1)
+}
+
+// RandomString generates a random string of length n
+func RandomString(n int) string {
+	var sb strings.Builder
+	k := len(alphabet)
+
+	for i := 0; i < n; i += 1 {
+		c := alphabet[rand.Intn(k)]
+		sb.WriteByte(c)
+	}
+
+	return sb.String()
+}
+
+//  RandomCategory generates a random owner name
+func RandomCategory() string {
+	return RandomString(6)
+}
+
+// RandomMoney generates a random amount of number
```

Comment:
Corrige o nome da função para mim, acabei esquecendo de mudar antes de te enviar :sweat_smile:
```suggestion
// RandomCode generates a random amount of number
```